Tools/Scripts/webkitpy/benchmark_runner/webserver_benchmark_runner.py

```diff
@@ -63,6 +62,8 @@ def _run_one_test(self, web_root, test_file, iteration):
         try:
             self._http_server_driver.serve(web_root)
             url = urljoin(self._http_server_driver.base_url(), self._plan_name + '/' + test_file + self._construct_subtest_url(self._subtests))
+            if '?' not in url:
+                url = url.replace('&', '?', 1)
```

Comment:
Could you tell me more about this change?

Reply:
This was originally omitted because of an oversight, since the JetStream2 entry point already has a parameter in it (`index.html?report=true`) we can just keep appending with `&` in between the parameters. However, since the MotionMark subtest entry point does not have a parameter already (`developer.html`), we need to replace the first `?` in the query with a `&`. This could have also been done in `_construct_subtest_url()`, but to me here makes more sense in case we ever want to include more parameters before the subtest ones in the future.